include/xalloc.h

```diff
@@ -146,4 +146,16 @@ static inline
 	return name;
 }
 
+#ifndef HAVE_EXPLICIT_BZERO
+static inline void
+  __attribute__((nonnull))
+  explicit_bzero(void *v, size_t sz)
+{
+	volatile unsigned char *p = v;
+
+	while (sz--)
+		*p++ = '\0';
+}
+#endif
+
```

Comment:
Good idea add fallback solution for explicit_bzero(), but I think xalloc.h is not the right place. We use this header file in command line utils, but no in libs (as err() is unvanted in libs). It would be better to move it to include/c.h.